bin/install_travis.sh

```diff
@@ -12,8 +12,13 @@ if [[ "${WITH_SAGE}" != "yes" ]]; then
     conda config --set always_yes yes --set changeps1 no;
     conda update -q conda;
     conda info -a;
-
-    conda create -q -n test-environment python="${PYTHON_VERSION}" pip cython sympy nose pytest;
+    CONDA_PKGS="pip cython sympy nose pytest"
+    if [[ "${SKIP_NUMPY}" == "yes" ]]; then
```

Comment:
I agree with respect to logic of naming the option, however it would make more sense to do extensive testing of NumPy because this is what we can expect the majority to use. So inverting the case I guess I would add "WITH_NUMPY=yes" to all test cases but one, is that alright?